sdk/anomalydetector/azure-ai-anomalydetector/src/samples/java/com/azure/ai/anomalydetector/MultivariateSample.java

```diff
@@ -193,37 +197,17 @@ public static void run(BinaryData trainBody, BinaryData beginInferBody) throws E
         InputStream fileInputStream = new FileInputStream("azure-ai-anomalydetector\\src\\samples\\java\\sample_data\\sync_infer_body.json");
```

Comment:
@srnagar Where do we recommend to store sample data?

Currently it is here azure-ai-anomalydetector/src/samples/java/sample_data

Should it be in azure-ai-anomalydetector/src/samples/resources ?